presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/GeometryUtils.java

```diff
@@ -28,6 +28,11 @@ private static double translateFromAVNaN(double n)
         return n < -1.0E38D ? (0.0D / 0.0) : n;
     }
 
+    public static double translateToAVNaN(double n)
```

Comment:
What is `AV` here? I see `AV` in other methods' names in this class, we should rename them.

Reply:
This is a copy-paste from `ESRI` library. Added a comment with an explanation.